static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -26,7 +26,7 @@ import checkMediator from 'common/modules/check-mediator';
 import addEventListener from 'lib/add-event-listener';
 import identity from 'common/modules/identity/api';
 import url from 'lib/url';
-import cookies from 'lib/cookies';
+import { remove, add } from 'lib/cookies';
```

Comment:
Not sure about this. I think I'd prefer still writing `cookies.remove ...` for readability, because the method names are so generic.